alpha.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, _date, _data, _parameters):
         self.__last_success_date = None
 
     def __call__(self, date, data):
-        portfolio = self.generate(date, data)
+        portfolio = self.generate(data)
```

Comment:
Wait, removing `date` field in Portfolio doesn't mean that we don't need to pass `date` to alpha.

Reply:
date is the only way the alpha can realize the current date.